docs/AI-for-security/usecase-landing-pg.asciidoc

```diff
@@ -7,4 +7,12 @@ The guides in this section describe example workflows for AI Assistant and Attac
 * <<attack-discovery-ai-assistant-incident-reporting>>
 * <<esql-queries-assistant>>
 
-For general information, refer to <<security-assistant, AI Assistant>> or <<attack-discovery, Attack discovery>>.
\ No newline at end of file
+For general information, refer to <<security-assistant, AI Assistant>> or <<attack-discovery, Attack discovery>>.
+
+[discrete]
+== Other AI-powered tools
+
+In addition to AI Assistant and Attack Discovery, {elastic-sec} provides several other AI-powered tools for specific use-cases. These include:
+
+* <<automatic-import>>: helps you quickly parse, ingest, and create ECS mappings for data from sources that don't yet have prebuilt Elastic integrations. This can accelerate your migration to {elastic-sec}, and help you quickly add new data sources to an existing SIEM solution in {elastic-sec}.
```

Comment:
```suggestion
* <<automatic-import>>: Helps you quickly parse, ingest, and create ECS mappings for data from sources that don't yet have prebuilt Elastic integrations. This can accelerate your migration to {elastic-sec}, and help you quickly add new data sources to an existing SIEM solution in {elastic-sec}.
```